internal/repository/repository.go

```diff
@@ -28,23 +28,30 @@ type Repository struct {
 	keyName string
 	idx     *MasterIndex
 	restic.Cache
+	minPackSize uint
 
 	treePM *packerManager
 	dataPM *packerManager
 }
 
 // New returns a new repository with backend be.
-func New(be restic.Backend) *Repository {
+func New(be restic.Backend, minpacksize uint) *Repository {
```

Comment:
That's probably the best solution, makes more sense to use bytes.  I'll go ahead and make that change, this will also resolve your concern in testing.go#103.